### PR TITLE
Fixed minimum height issue

### PR DESCRIPTION
**Pull request by levonpetrosyan93**
Fix for issue #1112

### CODE REVIEW

**Comment by justanwar** [comment]
Tested. Main wallet window minimum height reduced but recovery UI is still tall. 

Need testing on 1366x768 screens.

**Comment by levonpetrosyan93** [comment]
Tested on 1280x800

**Comment by justanwar** [comment]
Tested on 1280x768, mnemonic recovery screen shows fully.

File: src/sigma/test/serialize_test.cpp [hunk 1]
```diff
@@ -21,21 +21,10 @@ BOOST_AUTO_TEST_CASE(group_element_serialize)
...
-    for (int i = initial.memoryRequired() - 1; i > 1; i--)
-        buffer[i] = buffer[i - 2];
-    buffer[0] = std::rand();
-    buffer[1] = std::rand();
-
-    // Making the point not infinity as it is allowed in deserialization
-    buffer[33] = 0;
-
+    // Invalid GroupElement generated in advance
+    std::string buffer = "956f89850d79edd157360bb8608b8a336ca6e00dc057b4182540db5536745ef57b00";
```
Comment (AaronFeickert) [inline src/sigma/test/serialize_test.cpp:25]: Can you provide an explanation for how this particular invalid serialization was derived, and why it is suitable to test the desired functionality?

Comment (levonpetrosyan93) [reply src/sigma/test/serialize_test.cpp:25]: Previous test code sometimes failed there is possibility to generate valid point from such semi-random string, I generated an invalid point checked that, and put it here.

Comment (AaronFeickert) [reply src/sigma/test/serialize_test.cpp:25]: Sure, but what makes it a good test candidate? I'm of the opinion that magic numbers should be easily explainable or reproducible where possible.

**Comment by KuroGuo** [comment]
Tested on 1280x768 (Windows 11, zh-CN), mnemonic recovery screen shows fully.

image

**Pull request merged**